### PR TITLE
v0.5.0 dependency updates + changelog

### PR DESCRIPTION
**Pull request by chrissimpkins**
Updates project dependencies in preparation for the v0.5.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+- New: include compiler crashes in ICE regression definition
+- New: ANSI escape code colored standard stream output
+- New: More concise final report intended for copy/paste into rust-lang/rust issue tracker threads. This eliminates some of the Markdown elements that are not generally included in threads by users
+- New: Add bisect-rustc version to the final report
+- New: Add host triple to the final report
+- New: Add command line args to reproduce the reporter's bisect-rustc tests to final report
+- Fix: end date reporting when `--end` option used without `--start` option
+- Updated: Standard stream reporting format for improved readability during execution
+- Updated: Final report instructions for regression reporting
+- Updated:
```
Comment (bjorn3) [inline CHANGELOG.md]:
```suggestion
```

Comment (chrissimpkins) [reply CHANGELOG.md]: Thanks! Sorry I missed this.  I pushed https://github.com/rust-lang/cargo-bisect-rustc/pull/87/commits/a1af249541815c5c718a45467fbfe24a7940c469 with a few edits before you submitted this.

Comment (bjorn3) [reply CHANGELOG.md]: race-condition :)

Comment (chrissimpkins) [reply CHANGELOG.md]: ha!

**Pull request merged**